client/signup/test/utils.js

```diff
@@ -34,7 +34,13 @@ describe( 'utils', function() {
 
 	before( () => {
 		flows = require( 'signup/config/flows' );
+
 		sinon.stub( flows, 'getFlows' ).returns( mockedFlows );
+		sinon.stub( flows, 'preloadABTestVariationsForStep', ()=>{} );
```

Comment:
Was unsure for a moment on whether `() => {}` would return `{}` or `undefined` 😅